content/operate/kubernetes/upgrade/upgrade-olm.md

```diff
@@ -57,6 +57,28 @@ See [Upgrade modules]({{<relref "/operate/oss_and_stack/stack-with-enterprise/in
 
 Use `kubectl get rec` and verify the `LICENSE STATE` is valid on your REC before you start the upgrade process.
 
+## Upgrade with Helm charts
```

Comment:
Users installing via OLM and not installing via Helm, kinda by definition.
I think we don't need this section here within the OLM docs.